### ISSUE TITLE
Build task on windows fails due to OS path separator issue

### ISSUE THREAD

**Issue by msanders70** [krakenjs/makara-builder#1]
The grunt-makara-amdify 1.0.1 uses the makara-builder 1.0.0 and in makara-builder/index.js I can see:

```
var locales = paths.map(function (p) {
    var m = /(.*)\/(.*)/.exec(path.relative(localeRoot, p));
    return m[2] + '-' + m[1];
});
```

localeRoot="...\locales" and p="...locales/DE/de"

The path.relative() call transforms p into windows style and returns "DE\de".
The reg ex evaluation returns null because of the hardcoded linux path sep.
The following line gives of course "Fatal error: Cannot read property '2' of null".

Any hint for a workaround? Beside "use linux" ... ;-)

**Comment by grawk** [comment]
We will resolve this. And also add a set of unit tests for this module.. Thanks for bringing it to our attention

**Comment by grawk** [comment]
This should be resolved with `v1.0.1`
